src/Sulu/Bundle/MediaBundle/Content/Types/MediaSelectionContentType.php

```diff
@@ -200,10 +201,17 @@ public function preResolve(PropertyInterface $property)
     public function mapPropertyMetadata(ContentPropertyMetadata $propertyMetadata): PropertyMetadata
     {
         $mandatory = $propertyMetadata->isRequired();
-        $minItems = $mandatory ? 1 : 0;
+        $minMaxValue = SelectionPropertyMetadataMapper::getValidatedMinMaxValue($propertyMetadata);
```

Comment:
Sorry to say, but this feels wrong to me... If we want to reuse stuff, the Symfony way would be to use DI for that, not static methods... That introduces hard coupling, makes testing harder, etc. I think there is a reason we are using DI nowadays.